src/utils.js

```diff
@@ -15,9 +15,12 @@ export function compose(...funcs) {
 
 // Get the icon URL of an app
 export function appIconUrl(app) {
-  return app && app.baseUrl
-    ? resolvePathname('images/icon.svg', app.baseUrl)
-    : null
+  if (app && app.baseUrl && Array.isArray(app.icons)) {
+    const iconSize =
+      app.icons.find(({ sizes }) => sizes === '22x22') || app.icons[0]
```

Comment:
What would you think about doing something like this:

- Parse the sizes available in `app.icons` (`parseInt()` should be enough to get the width).
- Pass the size at which we want to render the icon to `appIconUrl()`, as an optional parameter (a single number is enough since icons are square).
- If a size is provided, send the closest icon which size is higher or equal.
- If no size is provided, or the closest size higher or equal can’t be found, send the largest icon available.